lib/features/cvm/cvm_middleware.dart

```diff
@@ -14,6 +15,7 @@ import 'package:redux/redux.dart';
 class CvmMiddleware extends MiddlewareClass<AppState> {
   final CvmFacade _facade;
   StreamSubscription<List<CvmMessage>>? _subscription;
+  bool _featureFlipConsumed = false;
```

Comment:
ça veut dire qu'il peut y avoir plusieurs FeatureFlipAction ?

Reply:
Très bon point !
En vrai, ce `_featureFlipConsumed`  est bien dégueu, et j'aurais aimé trouvé une autre manière de le faire…
Pour contre, si je suis d'accord avec ta proposition, on ne peut pas initialiser le feature flip CVM au lancement de l'app, sans que le jeune soit connecté, car il nous faut récupérer son conseiller, et pour cela il doit être connecté.

Du coup, j'ai fait un changement, de sorte que la FeatureFlipAction ne prenne pas le FF en entier, mais juste ces différents champs. Ce qui me permets dans le CvmMiddleware de faire un 
`bool _shouldStartCvm(dynamic action) {
    return action is CvmRequestAction || (action is FeatureFlipAction && action.useCvm);
  }`, et par la même supprimer le `_featureFlipConsumed`.

Tu me diras si ça te va :)